### PR TITLE
Update chrome ui

### PR DESCRIPTION
**Pull request by luu-alex**
Updated UI
Screen Shot 2019-11-03 at 10 55 44 PM
Screen Shot 2019-11-03 at 10 55 29 PM
Screen Shot 2019-11-03 at 10 55 24 PM

### CODE REVIEW

File: extension/src/popup.js [hunk 1]
```diff
@@ -94,66 +110,99 @@ stopToonin.onclick = () => {
...
     var activeTab = tabs[0];
     chrome.tabs.sendMessage(activeTab.id, {"message": "clicked_extension"});
 });
 
   chrome.runtime.onMessage.addListener(function(request, sender, sendResponse) {
     if (request.message === "extension_state_from_background" && request.data.roomID) {
-        roomNameSpan.style.visibility= "hidden";
-        shareButton.style.visibility= "hidden";
-        stopSharingButton.style.visibility="visible";
-        copyButton.style.visibility="visible";
```
Comment (alir128) [inline extension/src/popup.js]: Maybe you can send a call to the create room button function (line 113)

File: extension/src/popup.js [hunk 2]
```diff
@@ -94,66 +110,99 @@ stopToonin.onclick = () => {
...
-        peerCounter.style.visibility = "visible";
+        roomDiv.style.display = "block";
+        peerCounter.style.display = "block";
         peerCounter.innerHTML = "You have " + request.data.peerCounter + " listeners.";
-    } 
+        roomNameSpan.style.display = "none";
+        btnShare.style.display = "none";
+        backButton.style.display = "none";
+    }
     else if (request.message === "extension_state_from_background" && !request.data.roomID && request.data.playing) {
```
Comment (alir128) [inline extension/src/popup.js]: call to function connectRoomBtn (line 122) can be made and disable the create room functionalities.

File: extension/src/toonin_box.html [hunk 3]
```diff
@@ -53,7 +53,8 @@ <h2 class="mdl-card__title-text">Toonin
             </a>
             <p class="mdl-card__supporting-text room-id-text" id="roomID">
             </p>
```
Comment (alir128) [inline extension/src/toonin_box.html]: this file is useless and should be deleted.

File: extension/src/popup.js (hunk 1 above)
Comment (luu-alex) [reply extension/src/popup.js]: Hey thanks for the feedback adresssing them right now.
For 3, the stop button is visible when you connect to someone elses room.

**Comment by Lakshya2610** [comment]
@luu-alex I was testing the changes. 

1. The text fields need padding.
2. Reduce to font weight maybe? ( I personally like fonts with lighter weights, but that's pretty subjective)

3. The logo gets cut-off when on the streaming page

4. The rounded text fields look nice. But, can we make the border thinner?

**Comment by luu-alex** [comment]
@Lakshya2610 hey did most of the changes you suggested except I wasn't able to get 3. working, theres something strange with how our chrome extension and how it interacts with its classes. i'll fix it in another PR if its ok

**Pull request merged**